docs/administer/rbac.md

```diff
@@ -0,0 +1,498 @@
+
+        Example: To manage `backup schedules` (create, update, delete a schedule), it is recommended to **explicitly** grant `read` permissions for the `backup schedules` as well as `backup storages`.
+
+    === "Backups"
+    
+        For on-demand backups and schedules, you should grant `read` permissions for `backup storages` as well.
+
+
+    === "Restores"
+
+        For restores, to new and existing databases, you should grant the following permisssions as well:
+
+        - Read `backups` (of the old DB)
+        - Read `MonitoringConfig` (if the old DB has monitoring enabled)
+        - Read `database cluster credentials` (of the old DB)
+        - Create `backups` (if backup schedules are enabled)
+   
+    === "Upgrades"
+
+        For upgrades, the following permissions must be granted:
+
+        - Read Namespaces
+        - Read all `database engines` in that namespace
+        - Read all `database clusters` in that namespace
+        - Update `database clusters` in that namespace
+
+
+## Roles in RBAC
+
+In Role-Based Access Control (RBAC), a Role is a set of permissions that define what actions (like read, write, update, delete) can be performed on specific resources within Percona Everest. In RBAC, roles are assigned to users, allowing them to interact with the resources according to the permissions defined by their roles.
+
+!!! note
+    It is recommended to prefix role names with **role:** to prevent confusion between role names and user names.
+
+### Built-in role
+
+In Percona Everest, the only predefined role is **role:admin**. A user with this role has unrestricted access to Percona Everest. However, the RBAC (Role-Based Access Control) configuration can define and allocate specific roles based on individual requirements and access privileges.
+
+This default built-in role definition can be seen in [policy.csv](https://github.com/percona/everest/blob/main/deploy/quickstart-k8s.yaml#L94-L102).
+
+
+## RBAC examples
+
+In this section, we will explore some examples that demonstrate how to create policy definitions for the required roles.
+
+??? Example "Examples"
+
+    Let's dive into some role definitions for RBAC:
+    
+    === "Admin role" 
+    
+        **Admin role for a single namepsace**
+
+        Let's set up an admin role with unrestricted privileges to all the resources in a single namespace called `namespaceA`.
+
+        ```sh
+        p, role:namespaceAdmin, namespaces, *, namespaceA
+        p, role:namespaceAdmin, database-engines, *, namespaceA/*
+        p, role:namespaceAdmin, database-clusters, *, namespaceA/*
+        p, role:namespaceAdmin, database-cluster-backups, *, namespaceA/*
+        p, role:namespaceAdmin, database-cluster-restores, *, namespaceA/*
+        p, role:namespaceAdmin, database-cluster-credentials, *, namespaceA/*
+        p, role:namespaceAdmin, backup-storages, *, namespaceA/*
+        p, role:namespaceAdmin, monitoring-instances, *, namespaceA/*
```

Comment:
I'm so sorry, in the last review I suggested something that was wrong, you had it correctly at first.
```suggestion
        p, role:namespaceAadmin, namespaces, *, namespaceA
        p, role:namespaceAadmin, database-engines, *, namespaceA/*
        p, role:namespaceAadmin, database-clusters, *, namespaceA/*
        p, role:namespaceAadmin, database-cluster-backups, *, namespaceA/*
        p, role:namespaceAadmin, database-cluster-restores, *, namespaceA/*
        p, role:namespaceAadmin, database-cluster-credentials, *, namespaceA/*
        p, role:namespaceAadmin, backup-storages, *, namespaceA/*
        p, role:namespaceAadmin, monitoring-instances, *, namespaceA/*
```